include/message_filters/sync_policies/approximate_time.h

```diff
@@ -194,8 +194,8 @@ struct ApproximateTime : public PolicyBase<M0, M1, M2, M3, M4, M5, M6, M7, M8>
     else if ((msg_time - previous_msg_time) < inter_message_lower_bounds_[i])
     {
       RCUTILS_LOG_WARN_ONCE("Messages of type %d arrived closer ("
-        "%g ) than the lower bound you provided ("
-        "%g) (will print only once)",
+        "%ld ) than the lower bound you provided ("
```

Comment:
From what I can tell, this is printing out a `rclcpp::Time.nanoseconds()`, which is an int64_t.  While the above most likely works on 64-bit arches, the truly portable way to do this is with `%PRId64` (that way it won't warn on 32-bit either).  I'm going to change these two to use that instead.